.circleci/config.yml

```diff
@@ -0,0 +1,142 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+orbs:
+  docker: circleci/docker@1.5.0
+jobs:
+  heroku16pg96:
+    docker: &docker
+      - image: cimg/base:2020.01
+    environment:
+      STACK: heroku-16
+      POSTGRESQL_VERSION: 9.6
+    steps: &checkout_and_test
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - checkout
+      - run:
+          name: "Running test"
+          command: ./support/test.sh "${STACK}"
+
+  heroku16pg11:
```

Comment:
Not blocking: I think you could make this file slightly less verbose by using parameterized jobs instead of a bunch of reused config: https://circleci.com/blog/parameters-in-pipeline-config/.

Reply:
ill check this out